tests/jax/latest/flax-bart-wiki_summary.libsonnet

```diff
@@ -19,8 +19,8 @@ local tpus = import 'templates/tpus.libsonnet';
 {
   local hf_bart_common = common.JaxTest + common.huggingFace {
     local config = self,
-    frameworkPrefix: 'flax-latest',
-    modelName:: 'hf-bart',
+    frameworkPrefix: 'flax.latest',
+    modelName:: 'bart-wiki.summary',
```

Comment:
from a readability point of view, would it be advantageous to separate the dataset from the model name?

Reply:
Thanks! Yeah I know. I also mentioned in the description `Add dataset info into modelName config instead of adding a new field to avoid many if/else statement in Jsonnet config (other frameworks do not have this dataset info, i.e. PyTorch, PAX, etc)`. And the format is defined here: https://github.com/GoogleCloudPlatform/ml-testing-accelerators/blob/master/templates/base.libsonnet#L80

Reply:
ah I missed that, thank you